src/owl-core.xsl

```diff
@@ -36,7 +36,7 @@
     <xsl:import href="owl-core-lib/owl-core-selectors.xsl"/>
     <xsl:import href="owl-core-lib/connectors-owl-core.xsl"/>
     <xsl:import href="owl-core-lib/elements-owl-core.xsl"/>
-    <xsl:import href="../config/config-proxy.xsl"/>
+    <!-- xsl:import href="../config/config-proxy.xsl"/ -->
```

Comment:
how does this work without the config proxy? 
does not seem right. We need to discuss the meaning of that proxy perhaps.